docs/metrics.md

```diff
@@ -88,6 +94,38 @@ An event fired when the user tries to drag or drop a content into the notepad.
 - `cd5`
 - `cd6`
 - `cd10`
+- `cd11`
+
+#### `new-note`
+Event fired when a user creates a new note.
+- `ec` - `notes`
+- `ea` - `new-note`
+- `el` - one of `in-note` or `list-view` where `in-note` refers to the meatball menu flow and `list-view` refers to the button in the list view
+- `cd10`
+- `cd11`
+
+#### `export`
+An event fired when a user exports a note.
+- `ec` - `notes`
+- `ea` - `export`
+- `el` - one of `html`, `markdown`, `plaintext`, `richtext`
+- `cd10`
+- `cd11`
+
+### `delete-note`
+Event fired when a user deletes a note.
+- `ec` - `notes`
+- `ea` - `new-note`
+- `el` - one of `in-note` or `exit-blank-note` which occurs when the user navigates back from a new note without adding content
```

Comment:
We no longer delete note on exit event, but on change event if content is empty (that solve many issues with multi window syncing). So I guess I will rename `exit-blank-note` to `blank-note`